IncrementalSVDFastUpdate.C

```diff
@@ -184,7 +186,7 @@ IncrementalSVDFastUpdate::computeBasis()
 
        if (d_updateRightSV)
        {
-           Matrix* d_basis_right_new = new Matrix(d_num_rows_of_W, d_num_samples-1, false);
+           Matrix* d_basis_right_new = new Matrix(d_num_rows_of_W, d_num_samples-1, d_basis_right->distributed());
```

Comment:
@chldkdtn Hi Youngsoo. In this function, should we also remove the last row and column of d_S whenever a singular_value is below d_singular_value_tol?

Reply:
Never mind. It seems like it's not needed.